rollup.config.js

```diff
@@ -0,0 +1,104 @@
+const fs = require('fs');
+const babel = require('rollup-plugin-babel');
+const readPkgUp = require('read-pkg-up');
+const resolve = require('rollup-plugin-node-resolve');
+const json = require('rollup-plugin-json');
+const commonjs = require('rollup-plugin-commonjs');
+const postcss = require('rollup-plugin-postcss');
+const peerDeps = require('rollup-plugin-peer-deps-external');
+const getBabelPreset = require('@commercetools-frontend/babel-preset-mc-app');
+const builtins = require('rollup-plugin-node-builtins');
+const babelPluginImportGraphQL = require('babel-plugin-import-graphql');
+const postcssCustomProperties = require('postcss-custom-properties');
+const postcssCustomMediaQueries = require('postcss-custom-media');
+const postcssColorModFunction = require('postcss-color-mod-function');
+const postcssDiscardComments = require('postcss-discard-comments');
+const postcssImport = require('postcss-import');
+const postcssPresetEnv = require('postcss-preset-env');
+const postcssReporter = require('postcss-reporter');
+
+const { pkg } = readPkgUp.sync({
+  cwd: fs.realpathSync(process.cwd()),
+});
+
+const format = process.env.npm_lifecycle_event.split(':')[1];
```

Comment:
Could you add a comment to explain what this is?

Reply:
Sorry. Wrong button. 

When bundling for CJS we do not want to generate named exports in the JSON plugin. There seems to be no other way to read the CLI arguments passed to Rollup. This uses the npm lifecycle event which is the run script‘s name to deduct what format we are building. In that it is a convention to deduct it from the name. Ive also not found another way and did this on flopflip too. Maybe a comment would help in addition we could throw if we cant deduct a known format here to avoid hard to trace bundling errors later.